OffchainLabs/arb-token-bridge
# feat: remove token list cache

### Summary

Get rid of caching the token lists in the bridge UI to avoid bugs related to UI having stale information.

### Steps to test

- Token search and import functionalities should work as expected
- There shouldn't be any token-list being stored in session storage (key - `arbitrum:bridge:token-lists`)
- While the token lists are loading, there is also a small enhancement that shows a loading state in the Search Panel to avoid abrupt UX changes. 
<img width="200" alt="image" src="https://user-images.githubusercontent.com/7558499/211769882-d3d997d4-3734-43d8-b2f8-b82308cf5408.png">
